microfactory/microfactory.go

```diff
@@ -217,7 +274,7 @@ func (p *GoPackage) findDeps(config *Config, path string, allPackages *linkedDep
 			return false
 		}
 		return true
-	}, parser.ImportsOnly)
+	}, parser.ImportsOnly|parser.ParseComments)
```

Comment:
Not that I'm really that concerned about performance, but which part is it that implements "we're only parsing the comments before the import statement"?